Automattic/wp-calypso
# Color Schemes: Remove Laser Black

#### Changes proposed in this Pull Request

* No more Laser Black.

#### Testing instructions

* Switch to this PR and go to `/me/account/` on calypso.live. Laser Black should not appear in the list of color schemes.
